### PR TITLE
use long options for -c -z -b in start scripts

### PR DESCRIPTION
**Pull request by davecramer**

### CODE REVIEW

File: gpMgmt/bin/gpinitsystem [hunk 1]
```diff
@@ -1267,7 +1267,7 @@ CREATE_QD_DB () {
 		BUILD_PERFMON $GP_DIR
 		ERROR_CHK $? "create perfmon directories and configuration file" 1
 		LOG_MSG "[INFO]:-Starting the Master in admin mode" 1
-		export PGPORT=$GP_PORT;$PG_CTL -w -l $GP_DIR/pg_log/startup.log -D $GP_DIR -o "-i -p $GP_PORT -c gp_role=utility -M master -b 1 -C -1 -z 0 -m" start >> /dev/null 2>&1
+		export PGPORT=$GP_PORT;$PG_CTL -w -l $GP_DIR/pg_log/startup.log -D $GP_DIR -o "-i -p $GP_PORT -c gp_role=utility -M master --gp_dbid=1 --gp_contentid=-1 -z 0 -m" start >> /dev/null 2>&1
```
Comment (hlinnaka) [inline gpMgmt/bin/gpinitsystem]: What's this change? I don't see any difference on this line. Is github's viewer playing a trick on me?

File: gpMgmt/bin/gpinitsystem [hunk 2]
```diff
@@ -1267,7 +1267,7 @@ CREATE_QD_DB () {
...
 		RET_TEXT="`$PG_CTL status -D $GP_DIR`"
 		RUNNING=`$ECHO $RET_TEXT|$EGREP -c "not running|neither"`
```
Comment (hlinnaka) [inline gpMgmt/bin/gpinitsystem:1273]: Should change "-z 0" here too

**Comment by hlinnaka** [comment]
Great! Can you create a PR directly for the main gpdb repository, so that we can get this applied there?

**Comment by davecramer** [comment]
On 3 August 2016 at 16:38, Heikki Linnakangas notifications@github.com
wrote:

> Great! Can you create a PR directly for the main gpdb repository, so that
> we can get this applied there?

Hmmm the main gpdb repo still has the switches until you apply pg_upgrade.
I don't think it would work ?

Dave Cramer

File: gpMgmt/bin/gpinitsystem (hunk 1 above)
Comment (davecramer) [inline gpMgmt/bin/gpinitsystem]: Me neither I can try to get rid of it

**Comment by hlinnaka** [comment]
On 08/03/2016 11:39 PM, Dave Cramer wrote:

> On 3 August 2016 at 16:38, Heikki Linnakangas notifications@github.com
> wrote:
> 
> > Great! Can you create a PR directly for the main gpdb repository, so that
> > we can get this applied there?
> 
> Hmmm the main gpdb repo still has the switches until you apply pg_upgrade.
> I don't think it would work ?

The long options should work there too, no? Even if the short switches 
also exist.

Actually, you could also cherry-pick the removal of the clashing short 
switches into master. Would be good to get that done in master too, 
before trying to merge the pg_upgrade stuff.
- Heikki

**Pull request merged**